diff --git a/b/conf/bblayers.conf b/a/conf/bblayers.conf
index 49313a5..965021d 100644
--- a/b/conf/bblayers.conf
+++ b/a/conf/bblayers.conf
@@ -1,4 +1,4 @@
-LCONF_VERSION = "6"
+LCONF_VERSION = "7"
 
 BBPATH = "${TOPDIR}"
 BSPDIR := "${@os.path.abspath(os.path.dirname(d.getVar('FILE', True)) + '/../..')}"
@@ -29,3 +29,6 @@ BBLAYERS += " ${BSPDIR}/sources/meta-qt5 "
 #Embedded Artists Yocto layer
 BBLAYERS += " ${BSPDIR}/sources/meta-ea "
 BBLAYERS += " ${BSPDIR}/sources/meta-murata-wireless "
+#KEPM layer
+BBLAYERS += "${BSPDIR}/sources/meta-vf"
+BBLAYERS += "${BSPDIR}/sources/meta-python_st"
diff --git a/b/conf/local.conf b/a/conf/local.conf
index cf88bdb..d145356 100644
--- a/b/conf/local.conf
+++ b/a/conf/local.conf
@@ -4,7 +4,7 @@ PACKAGE_CLASSES ?= "package_rpm"
 EXTRA_IMAGE_FEATURES ?= "debug-tweaks"
 USER_CLASSES ?= "buildstats image-mklibs image-prelink"
 PATCHRESOLVE = "noop"
-BB_DISKMON_DIRS ??= "\
+BB_DISKMON_DIRS = "\
     STOPTASKS,${TMPDIR},1G,100K \
     STOPTASKS,${DL_DIR},1G,100K \
     STOPTASKS,${SSTATE_DIR},1G,100K \
@@ -17,5 +17,75 @@ PACKAGECONFIG_append_pn-qemu-native = " sdl"
 PACKAGECONFIG_append_pn-nativesdk-qemu = " sdl"
 CONF_VERSION = "1"
 
+BB_NUMBER_THREADS = '4'
+PARALLEL_MAKE = '-j 4'
+
 DL_DIR ?= "${BSPDIR}/downloads/"
 ACCEPT_FSL_EULA = "1"
+
+TOOLCHAIN_TARGET_TASK_append = " libpcap-dev"
+
+
+
+LOCALE_UTF8_ONLY = "1"
+LOCALE_UTF8_IS_DEFAULT = "1" 
+
+IMAGE_INSTALL_append = " \
+   wget \
+   nano \
+   kernel-modules \
+   kernel-devicetree \
+   gcc \
+   libstdc++ \
+   gcc-dev \
+   gdbserver \
+   openssh-sftp-server \
+   addlin \
+   addm4 \
+   addkalina \
+   addsettings \
+   tzdata \
+   startlin \
+   startrndis \
+   ptpd \
+   linuxptp \
+   python3 \
+   python3-spidev \
+   python3-pydbus \
+   glib-2.0 \
+   dbus \
+   libpcap \
+   libpcap-dev \
+   addksz8463 \
+   addhsr \
+   addvlan \
+   addwebserver \
+   startksz8463 \
+   ethtool \
+   iproute2 \
+   htop \
+   coreutils \
+   glibc-utils \ 
+   localedef \
+   procps \
+   sudo \
+   bash \
+"
+#X11
+DISTRO_FEATURES_append += " X11"
+EXTRA_IMAGE_FEATURES = " ssh-server-openssh debug-tweaks"
+#CORE_IMAGE_EXTRA_INSTALL += " firmware-imx-sdma"
+#KERNEL_MODULE_AUTOLOAD += "rpmsg_multiept"
+#KERNEL_MODULE_AUTOLOAD += "spi_loopback_test"
+
+# User/Group modifications
+# - Adding user 'kepm' with password and root permissions
+# - Setting password for user 'root' to 'passkepm'
+# - For more options see extrausers.bbclass
+INHERIT += " extrausers"
+
+EXTRA_USERS_PARAMS = " \
+  useradd -P 'kepm' kepm; \
+  usermod -s /bin/sh kepm; \
+  usermod -P 'pass_KEPM_2020' root; \
+"
